project/api/database/migrations/1573518080755_art_schema.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+/** @type {import('@adonisjs/lucid/src/Schema')} */
+const Schema = use('Schema')
+
+class ArtSchema extends Schema {
+  up () {
+    this.table('arts', (table) => {
+      table.string('duration', 80)
```

Comment:
I think it would be more appropriate if you put 16.

Reply:
Fixed #28